dateutil/test/test_isoparser.py

```diff
@@ -473,12 +482,11 @@ def test_isotime_midnight(isostr):
     ('14時30分15秒', ValueError),                # Not ASCII
     ('14_30_15', ValueError),                   # Invalid separators
     ('1430:15', ValueError),                    # Inconsistent separator use
-    ('14:30:15.3684000309', ValueError),        # Too much us precision
     ('25', ValueError),                         # Invalid hours
     ('25:15', ValueError),                      # Invalid hours
     ('14:60', ValueError),                      # Invalid minutes
     ('14:59:61', ValueError),                   # Invalid seconds
-    ('14:30:15.3446830500', ValueError),        # No sign in time zone
+    ('14:30:15.34468305:00', ValueError),       # No sign in time zone
```

Comment:
I wasn't quite sure about the original intent of this test case. The other case I see that says "No sign in time zone" has a `:` within the zone, but lacks a `-` or `+` prefix, so I followed suit here.

Reply:
Yes that is the right thing to do, though it does bring up an unfortunate ambiguity. Anyone who is trying to validate ISO8601 datetimes will silently accept `%Y-%m-%dT%H:%M:%S.%fhhmm`.

I think this is OK because there are other ways to generate valid ISO8601 strings that do not represent the intended datetime.